README.md

```diff
@@ -98,9 +98,13 @@ Register a factory function to create mock responses for each PATCH request to a
 
 Register a factory function to create mock responses for each DELETE request to a specific URL.
 
-#### .mock(method, url | regex, fn)
+#### .any(fn)
```

Comment:
This is my fault i missed that `.mock()` can configure MockXMLHttpRequest to handle any requests.

Because it's a little bit confusing how it was written (I'm not a Javascript adept). Maybe we can leave behavior you desined but create alias method if you like naming.